### ISSUE TITLE
Autofill Chrome Android not working

### ISSUE THREAD

**Issue by sjtinsley83** [bitwarden/mobile#562]
I am a new user of BitWarden. I want to used the autofill on my phone. I have enabled The autofill service but it doesn't work. Chrome is up to date and Android is version 9. Can someone help.

**Comment by kspearrin** [comment]
In chrome, navigate to "chrome://flags" and make sure autofill accessibility settings are turned on. Then restart and try again.

**Comment by sjtinsley83** [comment]
Unfortunately, that doesn't work. If I used the accessibility autofill mode it works, but it would like to use the better autofill framework that is in Android. It appears there are quite a few people experiencing the same issue.

**Comment by niikoo** [comment]
+1

The experience is very unstable. Mostly resolved by restating the browser though. But that's not a good solution.

**Comment by TimHeinrich** [comment]
Same here, but now it got even worse, not sure if the latest update of Chrome or Bitwarden broke it but now it does not even find the credentials anymore and I have to manually search for it. While still having the issue that I have to restart the browser. As a workaround I switched to the old autofill service

**Comment by niikoo** [comment]
@TimHeinrich I got the same problem. "com.android.chrome" is the identifier, not the webpage anymore.. Suddenly happened after the latest update. Version 2.2.0

**Comment by kspearrin** [comment]
We have not changed anything regarding autofill, so maybe there was a change in Chrome.

**Comment by krallus** [comment]
I am also new to Bitwarden and a little thrown by this behaviour.  Here's a way to consistently recreate the problem.  I'm using a Google Pixel 3a.  Android 9 (build PQ3B.190705.003). Chrome 76.0.3809.89 (last update July 30, 2019).  Bitwarden 2.2.0 (last update Aug 2, 2019).

1) Using Chrome in Android, browse to https://www.realtor.ca/
2) Click on the person icon and then sign-in
3) Click on the bitwarden notification (at top of Android screen)
4) Screen shows "Items for realtor.ca" -- this is correct.
5) Click on one of the fields and click "Auto-fill with Bitwarden"
6) Screen shows "Items for com.android.chrome" -- *** this is incorrect ***
7) Click the back button and press anywhere on the screen *not* on a field until the bitwarden notification re-appears back at the top of the screen
8) Click that Bitwarden notification and it is now again showing, correctly, "Items for realtor.ca" (but not always consistently .. see below)

Note that sometimes clicking on the top Bitwarden notification will also show "Items for com.android.chrome", but clicking back and checking the notification again will get it to go back to showing "Items for realtor.ca".

Clicking on a field and "Auto-fill with Bitwarden" *consistently* shows "Items for com.android.chrome".

At no point does a *load* of the realtor.ca sign-in screen ever get auto-populated with the Bitwarden-stored credentials, but opening up the notification while it correctly shows "Items for realtor.ca" and clicking on the stored credentials entry does populate the fields with the credentials.

I have Android's auto-fill service set to Bitwarden.  In Bitwarden's settings, both the "Auto-fill Service" and the "Auto-fill Accessibility Service" are showing as "Enabled".  The latter's "Always Scan" option is enabled.

I hope this helps.

**Comment by DougParker1992** [comment]
I'm also having autofill issues again. It's actually starting to get super annoying with it breaking over and over again...

**Comment by kspearrin** [comment]
@niikoo @DougParker1992 I just debugged the app and it appears that chrome is no longer sending the `WebDomain` property with their ViewNode objects. Therefore, I cannot detect the website of the autofill request when using the autofill service in Bitwarden. This appears to be a regression in Chrome. Other browsers, such as Firefox Focus, seem to still be working fine.

I am going to close this issue since we are tracking these various Chrome issues here: https://github.com/bitwarden/mobile/issues/489

**Comment by kspearrin** [comment]
I just tested some other password manager apps and it appears to be an issue there too. I don't think there is anything we can do here.

**Comment by ghhv** [comment]
Hi, 
@kspearrin An FYI.. I just started with Bitwarden .. getting the same issue on Chrome - BW is only detecting com.android.chrome but funnily enough my Lastpass which is still installed still works OK. (How Lastpass is still auto-filling when it's no longer set as the auto fill is also curious.. but I'm getting both..)
So did a reboot since I had just installed BW.. behaviour was unchanged. 
Disabled LastPass on Accessibility.. which stopped the LP fill.. but BW did nothing.. rebooted again..
BW is working again.. but again only matching com.android.chrome. 
I have autofill and accessibility enabled for BW. 
Running on Android 9 with Chrome 76.0.3809.111
Note: Auto-fill still works if I manually search for the login. 
I wish you success!
